### PR TITLE
move serialization utils out of IPython.utils

### PR DESCRIPTION
**Pull request by minrk**
- jsonutil to jupyter_client
- pickleutil to ipython_kernel

### CODE REVIEW

File: jupyter_client/tests/test_jsonutil.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 
 import nose.tools as nt
 
-from IPython.utils import jsonutil, tz
+from jupyter_client import jsonutil, tz
```
Comment (takluyver) [inline jupyter_client/tests/test_jsonutil.py]: This will break the tz import, I think - it should be moved to a separate line.

**Comment by takluyver** [comment]
codeutil is moved too, without a backwards compatibility shim. That's fine by me, but I wanted to make it clear.

File: jupyter_client/tests/test_jsonutil.py (hunk 1 above)
Comment (minrk) [inline jupyter_client/tests/test_jsonutil.py]: removed the use of tz by copying the relevant bits into the test file

**Comment by takluyver** [comment]
+1

**Pull request merged**